src/core/wee-network.c

```diff
@@ -96,23 +96,30 @@ void
 network_set_gnutls_ca_file ()
 {
 #ifdef HAVE_GNUTLS
-    char *ca_path, *ca_path2;
+    char *ca_path, *ca_path2, **single_paths;
+    int i;
 
     if (weechat_no_gnutls)
         return;
 
-    ca_path = string_expand_home (CONFIG_STRING(config_network_gnutls_ca_file));
-    if (ca_path)
+    single_paths = string_split(CONFIG_STRING(config_network_gnutls_ca_file), 
+                                PATH_SEPARATOR, 0, 0, NULL);
```

Comment:
I don't think it is a good idea to have the seperator OS dependent, if it doesn't have to be like here